cheroot/server.py

```diff
@@ -469,14 +523,14 @@ def _fetch(self):
         except ValueError:
             raise ValueError(
                 'Bad chunked transfer size: {chunk_size!r}'.
-                format(chunk_size=chunk_size),
-            )
+                    format(chunk_size=chunk_size),
+                    )
 
         if chunk_size <= 0:
             self.closed = True
             return
 
-#            if line: chunk_extension = line[0]
+        #            if line: chunk_extension = line[0]
```

Comment:
Please don't change formatting because it adds unnecessary noise to the PR. It's easier to review if only related bits show up in the diff.